doc/src/vpr/command_line_usage.rst

```diff
@@ -920,6 +980,82 @@ The following options are only valid when the placement engine is in timing-driv
 
     Name of the post-placement timing report file to generate (not generated if unspecfied).
 
+
```

Comment:
I know @Srivat97 has a PR that's about to land that documents the NoC options too. @Srivat97 : I suggest landing yours and then Sara can enhance any NoC documentation that doesn't capture everything she has here.